### PR TITLE
Add eyeson API

### PR DESCRIPTION
**Pull request by unused**
eyeson API is a service to handle video conferences with a [free plan for developers](https://apiservice.eyeson.com/api-pricing) providing 1k meeting minutes per month

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -589,6 +589,7 @@ Table of Contents
   * [synadia.com](https://synadia.com/ngs) — [NATS.io](https://nats.io) as a service. Global, AWS, GCP, and Azure. Free forever with 4k msg size, 50 active connections and 5GB of data per month.
   * [cloudkarafka.com](https://www.cloudkarafka.com/) - Free Shared Kafka cluster, up to 5 topics, 10MB data per topic and 28 days of data retention.
   * [pubnub.com](https://www.pubnub.com/) - Swift, Kotlin and React messaging at 1 million transactions each month. Transactions may contain multiple messages.
+  * [eyeson API](https://developers.eyeson.team/) - A video communication API service based on WebRTC (SFU, MCU) to build video platforms. Allows RealTime Data Injection, Video Layouts, Recordings, a fully featured hosted web UI (quickstart) or packages for custom UIs. Has a [free tier for developers](https://apiservice.eyeson.com/api-pricing) with 1000 meeting minutes a month.
```
Comment (ripienaar) [inline README.md:592]: nice, please list once only in the most appropriate catagory and resolve the conflict.

**Comment by unused** [comment]
thx, I just stumbled upon pubnub[1](https://github.com/ripienaar/free-for-dev/blob/master/README.md?plain=1#L591)[2](https://github.com/ripienaar/free-for-dev/blob/master/README.md?plain=1#L929) and thought that multiple entries are ok. eyeson is now listed in messaging. I also adapted the header to `Messaging and Streaming`.

**Comment by ripienaar** [comment]
You can remove pubnub dupe if you like, I must have missed it was a dupe :) Else happy to merge as is, sure someone else will fix

**Comment by unused** [comment]
merge would be nice thank you.

**Comment by ripienaar** [comment]
thanks

**Pull request merged**